options/locale/locale_pl-PL.ini

```diff
@@ -364,6 +364,7 @@ password_uppercase_one=Co najmniej jedna duża litera
 password_digit_one=Co najmniej jedna cyfra
 password_special_one=Co najmniej jeden znak specjalny (interpunkcja, nawiasy, cudzysłowy, itp.)
 enterred_invalid_repo_name=Wprowadzona nazwa repozytorium jest niepoprawna.
+enterred_invalid_org_name=Wprowadzona nazwa organizacji jest niepoprawna.
```

Comment:
```suggestion
```
will be overwritten by CI, pleace localize via Crowdin after pull got merged

Reply:
PL translation removed in 36a8157f76e5dc3a4f129a5d1af199e4007b33a4. Let me know if you need `enterred_invalid_org_name` removed from `locale_en-US.ini` also.

Reply:
No, eng is source of translation & is good as it is :)

Reply:
added translation for u here: https://crowdin.com/translate/gitea/68/enus-pl?filter=basic&value=2#q=enterred_invalid_org_name

Reply:
Translation is ok - thank you.